### DISCUSSION TITLE
Not currently working on listvertise

### DISCUSSION THREAD

**Discussion by jbooker1** [FastForwardTeam/FastForward#759] [Q&A] [answered]
As the title states, I downloaded fastforward for firefox and it wasn't able to work on listvertise.

**Comment by lostdusty** [comment, marked as answer]
You need to add [this script](https://github.com/FastForwardTeam/FastForward/blob/main/src/linkvertise.js) in custom bypass settings to linkvertise work on chrome.
See this announcement in our Discord server: 
![image](https://user-images.githubusercontent.com/47502554/210077133-537417fa-1352-44c3-b7dc-622abba7bd42.png)
